CityPuzzle/Game Room/Join GameRoom/SeeEnteredRooms.xaml.cs

```diff
@@ -0,0 +1,73 @@
+﻿using CityPuzzle.Classes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace CityPuzzle.Game_Room.Join_GameRoom
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class SeeEnteredRooms : ContentPage
+    {
+        public static List<Room> AllRooms = new List<Room>();
+
+
+        public SeeEnteredRooms()
+        {
+            InitializeComponent();
+            Task<List<Room>> treadFindRooms = new Task<List<Room>>(
+              () => {
+                  return GetData();
+              });
+            treadFindRooms.Start();
+            Task.WaitAll();
+            MyListView.ItemsSource = treadFindRooms.Result;
+
+
+        }
+        public static List<Room> GetData()
+        {
+            List<Room> findedRooms = new List<Room>();
+            Task <List<string>> treadFind = new Task<List<string>>(
+              () =>{
+                  return Sql.FindParticipantRoomsIDs(App.CurrentUser.ID);});
+            treadFind.Start();
+            Task tReadRooms = new Task(
+                () => {
+                    AllRooms = Sql.ReadRooms();
```

Comment:
This is because EntryGameRoomPage requires such a field. There are three ways to solve this:
1) Make it public- like now
2) IMake class EntryGameRoomPage from SeeEnteredRooms
3) SeeEnteredRooms - again read sql.
Which way is right for you?

Reply:
Keep it public. It was not clear where it is used as EntryGameRoomPage is not part of that commit